app/components/provider_interface/course_details_component.rb

```diff
@@ -12,10 +12,15 @@ def initialize(application_choice:)
       @provider_name_and_code = application_choice.provider.name_and_code
       @course_name_and_code = application_choice.course.name_and_code
       @cycle = application_choice.course.recruitment_cycle_year
-      @preferred_location = application_choice.site.name_and_code
+      @preferred_location = preferred_location_text
       @study_mode = application_choice.course_option.study_mode.humanize
     end
 
+    def preferred_location_text
+      "#{application_choice.site.name_and_code}\n" \
+      "#{address_formatter}"
```

Comment:
`address_formatter` feels like it's going to get an address and format it. `formatted_address`?

Reply:
Yes, good spot that is much clearer.